packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -382,7 +336,7 @@ class Form<T extends DbObject> extends Component<any,any> {
       document: this.state.initialDocument,
       name: fieldName,
       datatype: fieldSchema.type,
-      layout: this.props.layout,
+      layout: this.props.layout || "horizontal",
```

Comment:
That sure is a decision.

(I assume you're maintaining back-compat)

Reply:
This default was already present, using `(Form as any).defaultProps` at the bottom of the file. I'm not sure what this `layout` is used for or if it's even still used.

Reply:
I assume it's like, does the Form lay it's components out vertically or horizontally, and someone chose the wrong default according to me.